core/src/main/java/com/linecorp/armeria/server/metric/PrometheusExpositionService.java

```diff
@@ -35,20 +38,50 @@
  * Exposes Prometheus metrics in <a href="https://prometheus.io/docs/instrumenting/exposition_formats/">text
  * format 0.0.4</a>.
  */
-public final class PrometheusExpositionService extends AbstractHttpService {
+public final class PrometheusExpositionService extends AbstractHttpService implements TransientHttpService {
 
     private static final MediaType CONTENT_TYPE_004 = MediaType.parse(TextFormat.CONTENT_TYPE_004);
 
+    /**
+     * Returns a new {@link PrometheusExpositionService} that exposes Prometheus metrics from the specified
+     * {@link CollectorRegistry}.
+     */
+    public static PrometheusExpositionService of(CollectorRegistry collectorRegistry) {
+        return new PrometheusExpositionService(collectorRegistry, false);
+    }
+
+    /**
+     * Returns a new {@link PrometheusExpositionService} that exposes Prometheus metrics from the specified
+     * {@link CollectorRegistry}.
+     *
+     * @param shouldLogRequest tells whether the request and response are logged or recorded by
```

Comment:
How about: 
... whether the request and response handled by this {@link PrometheusExpositionService} are logged ...?